acceptance/setup/git/pre-suite/000_EnvSetup.rb

```diff
@@ -12,6 +12,7 @@
     'git',
     'ruby',
     'rubygem-json',
+    'rubygem-io-console'
```

Comment:
@Iristyle Do you mean is ```rubygem-io-console``` already in the image, or should we add it as part of the image creation ?

Should be no issue adding it if needed.